app/Jobs/Discord/SendNewFeature.php

```diff
@@ -26,9 +26,9 @@ class SendNewFeature implements ShouldQueue
      *
      * @return void
      */
-    public function __construct(Feature $feature)
+    public function __construct(int $feature)
     {
-        $this->feature = $feature->id;
+        $this->feature = $feature;
```

Comment:
I don't see $feature->image declared in the code anywhere (declared lower). Feature don't ever have a visible image